public/index.html

```diff
@@ -67,8 +67,8 @@ <h2>Our services</h2>
         </div>
         <h3>User-experience design</h3>
         <p>
-          User experience is the most important&mdash;but most neglected&mdash;aspect of web
-          development. We do user experience right, from start to finish
+          First we identify your users, their goals, and critical tasks. Then we design the
+          product they really need
```

Comment:
```suggestion
          We identify your users, their goals, and critical tasks in order to design the product that fits their needs.
```

Reply:
I prefer breaking it down into separate steps. The revised version leaves out the fact that we also do the design work.